pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -155,7 +155,13 @@ func (rc *reconciler) reconcile() {
 
 			// Before triggering volume detach, mark volume as detached and update the node status
 			// If it fails to update node status, skip detach volume
-			rc.actualStateOfWorld.RemoveVolumeFromReportAsAttached(attachedVolume.VolumeName, attachedVolume.NodeName)
+			err = rc.actualStateOfWorld.RemoveVolumeFromReportAsAttached(attachedVolume.VolumeName, attachedVolume.NodeName)
+			if err != nil {
+				glog.Errorf("RemoveVolumeFromReportAsAttached failed while removing volume %q from node %q with: %v",
```

Comment:
`glog.V(5).Infof` instead of `glog.Errorf` since the error is non-blocking. I want to prevent spamming logs so I'm cautious when adding new errors with high visibility to loops.